modules/contact/test/tests/bouncing-block-contact/grid-sequencing/no-grid-sequencing.i

```diff
@@ -0,0 +1,225 @@
+  [../]
+  [./disp_y]
+    block = '1 2'
+    # order = SECOND
+  [../]
+  [./normal_lm]
+    block = 3
+    # family = MONOMIAL
+    # order = CONSTANT
+  [../]
+  [./tangential_lm]
+    block = 3
+    family = MONOMIAL
+    order = CONSTANT
+  [../]
+[]
+
+[ICs]
+  [./disp_y]
+    block = 2
+    variable = disp_y
+    value = ${fparse starting_point + offset}
+    type = ConstantIC
+  [../]
+[]
+
+[Kernels]
+  [./disp_x]
+    type = MatDiffusion
+    variable = disp_x
+  [../]
+  [./disp_y]
+    type = MatDiffusion
+    variable = disp_y
+  [../]
+[]
+
+
+[Constraints]
+  [normal_lm]
+    type = NormalNodalLMMechanicalContact
+    slave = 10
+    master = 20
+    variable = normal_lm
+    master_variable = disp_x
+    disp_y = disp_y
+    ncp_function_type = min
+  [../]
+  [normal_x]
+    type = NormalMortarMechanicalContact
+    master_boundary = 20
+    slave_boundary = 10
+    master_subdomain = 4
+    slave_subdomain = 3
+    variable = normal_lm
+    slave_variable = disp_x
+    component = x
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+  [normal_y]
+    type = NormalMortarMechanicalContact
+    master_boundary = 20
+    slave_boundary = 10
+    master_subdomain = 4
+    slave_subdomain = 3
+    variable = normal_lm
+    slave_variable = disp_y
+    component = y
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+  [tangential_lm]
+    type = TangentialMortarLMMechanicalContact
+    master_boundary = 20
+    slave_boundary = 10
+    master_subdomain = 4
+    slave_subdomain = 3
+    variable = tangential_lm
+    slave_variable = disp_x
+    slave_disp_y = disp_y
+    use_displaced_mesh = true
+    compute_primal_residuals = false
+    contact_pressure = normal_lm
+    friction_coefficient = .1
+    ncp_function_type = fb
+  []
+  [tangential_x]
+    type = TangentialMortarMechanicalContact
+    master_boundary = 20
+    slave_boundary = 10
+    master_subdomain = 4
+    slave_subdomain = 3
+    variable = tangential_lm
+    slave_variable = disp_x
+    component = x
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+  [tangential_y]
+    type = TangentialMortarMechanicalContact
+    master_boundary = 20
+    slave_boundary = 10
+    master_subdomain = 4
+    slave_subdomain = 3
+    variable = tangential_lm
+    slave_variable = disp_y
+    component = y
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+[]
+
+[BCs]
+  [./botx]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 40
+    value = 0.0
+  [../]
+  [./boty]
+    type = DirichletBC
+    variable = disp_y
+    boundary = 40
+    value = 0.0
+  [../]
+  [./topy]
+    type = FunctionDirichletBC
+    variable = disp_y
+    boundary = 30
+    function = '${starting_point} * cos(2 * pi / 40 * t) + ${offset}'
+  [../]
+  [./leftx]
+    type = FunctionDirichletBC
+    variable = disp_x
+    boundary = 50
+    function = '1e-2 * t'
+  [../]
+[]
+
+# [Problem]
+#   num_grid_sequences = 3
```

Comment:
Is it possibly to just set `num_grid_sequences = 0` to deactivate grid sequencing? So that we do not need to duplicate input files.

Reply:
That means: we can get rid of this input file